app/models/views/overview/application.rb

```diff
@@ -46,6 +46,11 @@ def total_monthly_income
         @application.income ? format_currency(@application.income.round) : format_threshold_income
       end
 
+      def total_monthly_income_from_evidence
+        return nil if @application.evidence_check.blank?
+        format_currency(@application.evidence_check.income.round)
```

Comment:
I wonder why the income amount needs to be rounded?

Reply:
@leoapost I guess they don't want to deal with nickles.